presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1111,6 +1112,7 @@ std::string PrestoServer::getBaseSpillDirectory() const {
 void PrestoServer::enableRuntimeMetricReporting() {
```

Comment:
Let's rename this method to enableStatsReporting. Runtime metrics means different thing in Presto context

Reply:
@tanjialiang I also feel the common folder should contain core components and not optional components.

Reply:
Yeah this is indeed confusing. Checkout velox/common/base/RuntimeMetrics.h. It means some per query level run time stats that is exported along with the query. 
Maybe we should in this PR, at least make it consistent across codebase. Otherwise looking at the code alone, it will be super confusing. Then we can discuss to change the doc?

Reply:
>  I also feel the common folder should contain core components and not optional components.

That makes sense to me. Thanks Deepak